### PR TITLE
Log formatter

### PR DESCRIPTION
**Pull request by pappz**
## Describe your changes

Add human readable log output. It print out the exact source code line information.

### Checklist
- [ ] Is it a bug fix
- [ ] Is a typo/documentation fix
- [x] Is a feature enhancement
- [ ] It is a refactor
- [ ] Created tests that fail without the change (if possible)
- [ ] Extended the README / documentation, if necessary

### CODE REVIEW

**Review by pascal-fischer — APPROVED** [review]

**Review by mlsmaycon — DISMISSED** [review]

**Comment by pascal-fischer** [comment]
Will write a small test for `Format` and `parseSrc` functions.

File: formatter/formatter_test.go [hunk 1]
```diff
@@ -0,0 +1,26 @@
+package formatter
+
+import (
+	"github.com/sirupsen/logrus"
```
Comment (pappz) [inline formatter/formatter_test.go]: Please organize the imports.
https://bongnv.com/blog/2022-03-09-why-group-imports-in-go/

File: formatter/hook_test.go [hunk 2]
```diff
@@ -0,0 +1,38 @@
+package formatter
+
+import (
+	"github.com/stretchr/testify/assert"
```
Comment (pappz) [inline formatter/hook_test.go]: Please organize the imports.
https://bongnv.com/blog/2022-03-09-why-group-imports-in-go/

File: formatter/formatter_test.go (hunk 1 above)
Comment (pascal-fischer) [reply formatter/formatter_test.go]: Thanks, I did not know that. I just let the IDE write the imports. I pushed a fix ordering the imports

File: formatter/hook_test.go (hunk 2 above)
Comment (pascal-fischer) [reply formatter/hook_test.go]: Fixed

**Pull request merged**